ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml

```diff
@@ -891,6 +891,23 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-disconnected
+- as: azure-ipi-disconnected-p3-f28-long-duration-destructive
```

Comment:
Should be `azure-ipi-disconnected-p3-f28-long-duration-cloud`

Reply:
done